.eslintrc

```diff
@@ -13,7 +13,29 @@
     "react"
   ],
   "ecmaFeatures": {
-    "jsx": true
+    "jsx": true,
+    "arrowFunctions": true,
+    "binaryLiterals": true,
+    "blockBindings": true,
+    "classes": true,
+    "defaultParams": true,
+    "destructuring": true,
+    "forOf": true,
+    "generators": true,
```

Comment:
@hx-markterry There are indeed a whole bunch I don't want us using - that's not to say we should lint against them, there might be some exceptional use cases, but as far as daily coding goes, we need to be standing by our best practices. @Joezo is putting forward the bits he feels strongly about, then if we're all in agreement we'll whitelist those features and train/support our dev team in implementing them. If someone uses a feature that isn't mentioned in the best practices doc, expect it to get slammed in a PR (eg generators, let, es6 module syntax) just like any other bad practice.